KnockKnock-iOS/Entity/Challenge.swift

```diff
@@ -12,3 +12,8 @@ struct Challenges: Codable {
   let content: String
   let title: String
 }
+
+struct ChallengeTitle: Codable {
```

Comment:
```suggestion
struct ChallengeTitle: Decodable {
```
저희는 Decode 용으로 Entity를 사용하고 있으니 Decodable 이 더 명확해 보입니다! 다른것도 변경 부탁드려요